hbase-common/src/main/java/org/apache/hadoop/hbase/io/compress/Compression.java

```diff
@@ -352,6 +358,30 @@ public CompressionCodec reload(Configuration conf) {
           return lzmaCodec;
         }
       }
+    },
+    BROTLI("brotli", BROTLI_CODEC_CLASS_KEY, BROTLI_CODEC_CLASS_DEFAULT) {
```

Comment:
Consider adding `@SuppressWarnings("ImmutableEnumChecker")` -- I think this will quiet ErrorProne related to this lazy initializer.